src/ion/merge.rs

```diff
@@ -132,7 +134,8 @@ impl<'a, F: Function> Env<'a, F> {
             // `to` bundle is empty -- just move the list over from
             // `from` and set `bundle` up-link on all ranges.
             trace!(" -> to bundle{} is empty; trivial merge", to.index());
-            let list = core::mem::replace(&mut self.bundles[from].ranges, smallvec![]);
+            let vc = LiveRangeList::new_in(self.ctx.bump());
```

Comment:
Rather than `vc`, could we call this something more descriptive like `empty_vec`? (And below?)